### ISSUE TITLE
Email text image on player profile wrong colour in dark mode

### ISSUE THREAD

**Issue by TimothyJones** [kestasjk/webDiplomacy#703]
# Bug report

When your webdip display mode is set to dark and you look at a player profile with a public email address (eg http://webdiplomacy.net/profile.php?userID=36106), then the email address is displayed in light mode colours, eg:

<img width="267" alt="Screen Shot 2020-09-24 at 4 07 04 pm" src="https://user-images.githubusercontent.com/914369/94107055-087e8600-fe80-11ea-8f4a-6549491d00c6.png">

## Expected behaviour

The email should display in dark mode colours when it's dark mode.

## Actual behaviour

The email displayed in light mode colours. I'm repeating myself here. Who wrote this bug report template? Oh, I did. Hrm.\

## Steps to reproduce

1. Make a cup of tea
2. Set webdip to dark mode
3. Go to a profile page with a public email (eg http://webdiplomacy.net/profile.php?userID=36106 )
4. Observe that the email displays in light mode colours
5. Drink the tea

I'm not sure if the tea is necessary to reproduce the bug, I'm just telling you what I did.

**Comment by Claesar** [comment]
I can reproduce this on Edge and Mozilla. Doesn't happen on Chrome though.

**Comment by TimothyJones** [comment]
I experienced it on Chrome.

I think the images are cached on the server after they're generated, so probably it depends who viewed the link the first time?

**Comment by Claesar** [comment]
No, because I followed your link with Chrome (didn't see it) then Edge (saw the same as you) and Mozilla (same).

**Comment by PeregrineFlight** [comment]
If it possible to set the background as transparent? Even on the light-mode, it shows up with a white box in the grey page.

**Comment by TimothyJones** [comment]
Yes. This is the line that controls the colours: https://github.com/kestasjk/webDiplomacy/blob/master/profile.php#L826

You could use `imagecolourallocatealpha`, as here: https://www.php.net/manual/en/function.imagecolorallocatealpha.php

There are a couple easy solutions:

* Create two cached images, one for each visual mode.
* Just put the email addresses in the text of the site. Protecting emails from being scraped is less important nowadays.

Claesar didn't experience the problem in Chrome because he's a moderator, and moderators don't see the image.
